src/chat/chatter.py

```diff
@@ -175,18 +175,14 @@ def __ge__(self, other):
 
     def __lt__(self, other):
         """ Comparison operator used for item list sorting """
-        firstStatus = self.getUserRank(self)
-        secondStatus = self.getUserRank(other)
-
-        if self._me.player is not None:
-            if self.user.name == self._me.player.login:
-                return True
-            if other.user.name == self._me.player.login:
-                return False
+        if self.user.name == self._me.player.login:
+            return True
+        if other.name == self._me.player.login:
+            return False
 
         # if not same rank sort
-        if firstStatus != secondStatus:
-            return firstStatus < secondStatus
+        if self.user.rank != other.user.rank:
```

Comment:
User has no rank, probably didn't survive rebase. I guess this is the key part of fixing sorting.